### PR TITLE
improve description of what is backed up in custom scripts documentation

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: content/docs/snapshots/custom-scripts.md [hunk 1]
```diff
@@ -9,7 +9,13 @@ index: "other"
 
 {{< linked_headline "Custom Backup Commands" >}}
 
-Snapshots include customer console configuration, data from bind mounted volumes of all containers and if the customer instance is a multi-host instance, docker registry data will be backed up as well. You also have the ability to specify a custom script that will be run at the time of a backup. This script will run on the host that is running the primary Replicated container, not inside any of your containers. If you need this script to execute something within a container, you can call [Admin Commands](/docs/packaging-an-application/admin-commands/).
+Snapshots will always include customer console configuration and replicated settings.
```
Comment (divolgin) [inline content/docs/snapshots/custom-scripts.md]: `replicated settings` -- I think this needs to be specific.  When snapshot is restored, we don't restore things like private and public IP addresses, snapshot configuration, anything else instance specific?

File: content/docs/snapshots/custom-scripts.md [hunk 2]
```diff
@@ -9,7 +9,13 @@ index: "other"
...
+When not excluded with `exclude_registry_data: true`, docker registry data will be backed up as well.
+On the Native scheduler, all bind mounts will be backed up.
+For the [Swarm](/docs/snapshots/swarm) and [Kubernetes](/docs/snapshots/kubernetes/) schedulers, any desired volumes must be specified in your application yaml.
+You also have the ability to specify a custom script that will be run at the time of a backup.
```
Comment (divolgin) [inline content/docs/snapshots/custom-scripts.md]: No second person pronouns

File: content/docs/snapshots/custom-scripts.md [hunk 3]
```diff
@@ -9,7 +9,13 @@ index: "other"
...
+This script will run on the host that is running the primary Replicated container, not inside any of your containers.
```
Comment (divolgin) [inline content/docs/snapshots/custom-scripts.md]: Pretty sure script will run in replicated container and can take advantage of admin commands which run in their designated containers.

File: content/docs/snapshots/custom-scripts.md [hunk 4]
```diff
@@ -9,7 +9,13 @@ index: "other"
...
+If you need this script to execute something within a container, you can call [Admin Commands](/docs/packaging-an-application/admin-commands/).
```
Comment (divolgin) [inline content/docs/snapshots/custom-scripts.md]: no second person pronouns

File: content/docs/snapshots/custom-scripts.md (hunk 1 above)
Comment (laverya) [reply content/docs/snapshots/custom-scripts.md]: I think this should probably just mention console config, on reflection

File: content/docs/snapshots/custom-scripts.md (hunk 4 above)
Comment (laverya) [reply content/docs/snapshots/custom-scripts.md]: _But I didn't even change that line_ ☹️ 
I'll fix it

**Pull request merged**